src/xcode/ENA/ENA/Source/HTTPClient/Service/Services/CachedRestService.swift

```diff
@@ -131,4 +163,6 @@ class CachedRestService: Service {
 
 	private let optionalSession: URLSession?
 	private var cache: KeyValueCaching
+	private var fakeClientCacheDate: Date?
```

Comment:
Hm this name is confusing me and somehow I don't quite understand what its used for.
Somehow it gets initialised from the init that is only available in the fake but then it gets checked from the production code. Feels kind a odd.